src/main/java/com/dev/cinema/model/dto/request/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,40 @@
+package com.dev.cinema.model.dto.request;
+
+public class MovieSessionRequestDto {
+    private Long id;
```

Comment:
do we need this field?

Reply:
removed `id` from `MovieSessionRequestDto`